src/portal-components/DetailsPage/_DetailsPage.scss

```diff
@@ -155,17 +154,25 @@ $svg-icon-height: 30px;
     }
   }
 
-  .tab-content .tab-groups {
-    height: 55px;
-    box-shadow: 0px 5px 6px -4px rgb(0 0 0 / 15%);
-    @include SrcMixins.underline-tabs($inactive-color: unset);
-    justify-content: center;
-    .tab-item {
-      font-weight: 700;
-      flex-grow: 0.2;
-      border-radius: 0px;
-      color: unset;
-      background-color: unset;
+  .tab-content {
+    [role=heading] {
+      padding-top: 30px;
+    }
```

Comment:
This does put other headings at risk, but the way the DOM is currently set up for these components in the detail pages  it would need to be
```
    > * > * > [role=heading],
    > * > * > * > * > [role=heading],
    .SubsectionRowRenderer [role=heading]
```
I don't think this is much better